planty/posts/views.py

```diff
@@ -1,13 +1,14 @@
-from django.shortcuts import render
-from django.views.generic import CreateView, DetailView, UpdateView, DeleteView
-from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
-from .models import Post
-from django.urls import reverse_lazy
-from django.db.models import Q, Max, Min
-from django.utils import timezone
 from datetime import timedelta
+from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
```

Comment:
NIT: whitespace here between standard Python library imports and 3rd party imports (django)